contactsummary/js/src/contact-summary/services/contract.service.js

```diff
@@ -174,6 +175,22 @@ define([
     }
 
     /**
+     * Returns the contract field options
+     *
+     * @param  {string} fieldName
+     * @return {Promise}
```

Comment:
Why this function need to return a promise?

Reply:
Thank you for noticing it, I just realized it does not have to return a promise.